src/v/config/configuration.cc

```diff
@@ -3707,6 +3707,12 @@ configuration::configuration()
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       std::chrono::milliseconds(5ms),
       {.min = 5ms /* I guess? idk. */})
+  , iceberg_catalog_base_location(
+      *this,
+      "iceberg_catalog_base_location",
```

Comment:
nit: maybe worth mentioning that this shouldn't be changed after being set.

Reply:
ah, yeah will do.

Reply:
q: is it more like "shouldn't be changed after iceberg is enabled"? or am i misunderstanding the constraint?

Reply:
Yea that's apt -- the real constraint is that once topics are created that use the file system catalog, it'll be really bad if this config changes because it will look to redpanda as if all the metadata vanished

Reply:
done